src/main/java/io/github/thebusybiscuit/slimefun4/implementation/listeners/ClimbingPickListener.java

```diff
@@ -0,0 +1,41 @@
+package io.github.thebusybiscuit.slimefun4.implementation.listeners;
+
+import org.bukkit.GameMode;
+import org.bukkit.Sound;
+import org.bukkit.entity.Player;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import org.bukkit.util.Vector;
+
+import io.github.thebusybiscuit.slimefun4.api.events.ClimbingPickLaunchEvent;
+import io.github.thebusybiscuit.slimefun4.implementation.items.tools.ClimbingPick;
+import me.mrCookieSlime.Slimefun.Objects.SlimefunItem.SlimefunItem;
+import me.mrCookieSlime.Slimefun.SlimefunPlugin;
+
+/**
+ * The special event listener for {@link ClimbingPick}.
+ * This is made like this to allow addons to add more materials to climbing pick.
+ *
+ * @author Linox
+ *
+ * @see ClimbingPick
+ *
+ */
+public class ClimbingPickListener implements Listener {
```

Comment:
What the hell is the point of this listener...

Reply:
To allow for other addons to add more materials before actually applying the new velocity.

Reply:
This wont do shit for other addons though???? They can listen if they want. You doing this does literally nothing